src/resolvers/project.rs

```diff
@@ -68,4 +87,11 @@ impl Project {
             .load::<User>(conn)
             .expect("Could not load contributors")
     }
+
+    pub fn media(&self, conn: &diesel::PgConnection) -> Vec<Medium> {
+        media::table
+            .filter(media::project_id.eq(self.id))
+            .load::<Medium>(conn)
+            .expect("Could not load media")
```

Comment:
Is a `.expect` ok here? I'm guessing this is code that won't be in the release version, but I wanna make sure I'm not missing anything. Also, maybe we should flag this kind of thing? Something like `// @fix_later`

Reply:
Eh, I suppose we could do a Result but I'm fine with this being a fatal error.